packages/darklang/stdlib/html.dark

```diff
@@ -64,8 +65,8 @@ module Darklang =
 
           if (Stdlib.List.``member`` voidElements tag.name) then
             match attributesText with
-            | "" -> $"<{tag.name}/>"
-            | text -> $"<{tag.name} {text}/>"
+            | "" -> $"<{tag.name}>"
```

Comment:
I updated this from `<{tag.name}/>` to just `<{tag.name}>` since HTML doesn't recognize self-closing tags, and the parser disregards the slash character [link](https://developer.mozilla.org/en-US/docs/Glossary/Void_element#:~:text=If%20a%20trailing%20/%20(slash)%20character%20is%20present%20in%20the%20start%20tag%20of%20an%20HTML%20element)